core/src/main/java/tc/oc/pgm/wool/MonumentWoolFactory.java

```diff
@@ -123,7 +123,10 @@ public boolean isObjectiveWool(ItemStack stack) {
   }
 
   public boolean isObjectiveWool(org.bukkit.material.MaterialData material) {
-    return material instanceof Wool && ((Wool) material).getColor() == this.color;
+    // On 1.21, CraftItemStack#getData() on wool item stacks returns unsubclassed MaterialData,
+    // hence we're comparing the item type and color manually
+    return material.getItemType() == Materials.WOOL
```

Comment:
For 1.13+ the Wool material is always in the format `COLOR_WOOL`. This won't work for modern versions.

Reply:
it will work because we are manually defining the wools with legacy wool, but it's not something we should do anyways